### ISSUE TITLE
Support for yarn 2 pnp

### ISSUE THREAD

**Issue by smcenlly** [wallabyjs/public#2488]
Currently, Wallaby does not support yarn 2 pnp.

**Comment by nnennajohn** [comment]
Thanks for creating this. Following.

**Comment by smcenlly** [comment]
We spent the last 3 days completing our investigation on how to add yarn2 support to Wallaby.

We were hoping we would be able to implement this in a couple of days but it looks like it will take at least 1-2 weeks for us to get this working properly. 

We have existing short term commitments/plans that we need to complete before starting this work. We expect to know where this will fit on our schedule this in about 3-4 weeks time.

If you're reading this comment, please +1 the issue description to let us know you're interested in this feature (and have others you know who are interested do so as well).

**Comment by nnennajohn** [comment]
Thank you.

**Comment by smcenlly** [comment]
Closing this as a duplicate of https://github.com/wallabyjs/public/issues/2351. Will share most recent update in #2351.

**Comment by smcenlly** [comment]
Please see the [latest comment in #2351](https://github.com/wallabyjs/public/issues/2351#issuecomment-723930744) in getting yarn2 working with Wallaby

**Comment by nnennajohn** [comment]
Thank you.
